abci/types/types.proto

```diff
@@ -200,27 +200,21 @@ message ResponseCommit {
 // that can be adjusted by the abci app
 message ConsensusParams {
   BlockSize block_size = 1;
-  TxSize tx_size = 2;
-  BlockGossip block_gossip = 3;
+  EvidenceParams evidence_params = 2;
 }
 
 // BlockSize contains limits on the block size.
 message BlockSize {
+  // Note: must be greater than 0
```

Comment:
Couldn't it be -1?

Reply:
Unlimited? Do you think it's a good idea?

Reply:
Probably best if we're consistent, so these ints all have the same domain